content/hi/docs/reference/glossary/kubeadm.md

```diff
@@ -0,0 +1,18 @@
+---
+title: कुबेअदम (Kubeadm)
```

Comment:
```suggestion
title: क्यूबएडीएम  (Kubeadm)
```

Reply:
^ I am not sure if we should be phonetically translating it or otherwise. We pronounce it the way I have translated above.

Reply:
Yes, correct. Also, we can add this to the google sheet, it's not there

![Screenshot from 2022-10-27 00-56-37](https://user-images.githubusercontent.com/51878265/198119832-0642cdc7-9dce-4592-8112-8e124b2ba209.png)